toplevel/coqloop.ml

```diff
@@ -292,6 +278,9 @@ let coqloop_feed (fb : Feedback.feedback) = let open Feedback in
   | FileDependency (_,_) -> ()
   | FileLoaded (_,_) -> ()
   | Custom (_,_,_) -> ()
+  (* Re-enable when we switch back to feedback-based error printing *)
```

Comment:
Oh, coqc does generate feedback AFAICS. Note that this is also used in `coqtop`. You can just enable it and see for yourself. So the dilemma is like this: if we print errors using feedback, due to bug#5479 we miss some important errors. If we print using exceptions, we must ignore error feedback. Given the current state of affairs, the second choice is the one that makes sense.